rfcs/2021-07-19-8216-multiple-pipelines.md

```diff
@@ -257,6 +254,10 @@ Doesn't block to create other sources/sinks.
 
 ## Outstanding Questions
 
+1. Will components defined in the top-level configuration be considered part of a "default" pipeline or no pipeline at all? Sources and sinks seems straightforward to leave out of pipelines, but transforms are a more interesting question. They're the most obvious way to implement filters that limit the data available to specific pipelines.
+1. Is the concept of routes worth introducing, or should we allow an `outputs` option on transforms within a pipeline instead? Routes as described here are simple output connections, quite different from the same concept from competitors where they act as input definitions. If we think we'll want to add that later it may be worth saving the name.
```

Comment:
I don't have a strong opinion about this. It was a thing I considered initially but got convinced by @binarylogic 🤔 
I don't see anything that `routes` allow to do more than _just_ adding `outputs` on transforms. So to keep things simple, I'll change to `outputs`.

Reply:
I also do not have a strong opinion. I proposed routes so that pipelines could dictate where output flows, something that was missing from the original proposal. I thought routes might be more forward thinking, allowing us to add options to routes (like sampling and filtering shortcuts). I'm ok calling it something different.

Reply:
My strongest argument against routes is that we're more likely to add things like filtering to the inputs of pipelines than the outputs (probably?), so if we're going to introduce a new concept it'd be better if it covered either just inputs or both. But on the other hand, it could be slightly confusing to allow `outputs` on transforms only when defined within a pipeline.

Given that we're not really sure, I'd lean towards the simpler option that doesn't involve introducing a new concept.

Reply:
I agree with @lukesteensen, we'd probably do the filtering of a pipeline in the `inputs`, or even before reaching the pipeline itself, like inside the root configuration.
Let's move on, take the `outputs` option (unless someone is strongly against it).
I updated the RFC accordingly.